verticaltabs.jsm

```diff
@@ -465,6 +500,47 @@ VerticalTabs.prototype = {
     });
   },
 
+  clearFind: function () {
+    this.document.getElementById('find-input').value = '';
+    this.filtertabs();
+  },
+
+  filtertabs: function () {
+    let document = this.document;
+    let tabs = this.document.getElementById('tabbrowser-tabs');
+    let find_input = this.document.getElementById('find-input');
+    let input_value = find_input.value.toLowerCase();
+    let hidden_counter = 0;
+    let filterable_tabs = tabs.querySelectorAll('tab[verticaltabs-id]');
+    let hidden_tab = this.createElement('tab', {'class': 'tabbrowser-tab',
```

Comment:
One more thing…  Instead of creating this in code (and then having to remove it in code), can we put 
`<tab class="tabbrowser-tab" id="filler-tab" fadein="true" disabled="true"/>` at an appropriate place in `vertical-tabbrowser.xml`, and let the overlay stuff create and delete it for us?